jerry-core/api/jerry-snapshot.h

```diff
@@ -41,7 +41,7 @@ typedef struct
 /**
  * Jerry snapshot format version.
  */
-#define JERRY_SNAPSHOT_VERSION (18u)
+#define JERRY_SNAPSHOT_VERSION (19u)
```

Comment:
Since the snapshot version number has increased, [this](https://github.com/jerryscript-project/jerryscript/blob/master/tests/unit-core/test-snapshot.c#L226) expected output should be modified as well.

Reply:
Is this really a change in the snapshot version at all?

Reply:
@rerobika @akiss You are right, the expected output of the snapshot test also should be modified. Thanks.

Reply:
The format of snapshots are not changed in this patch. I agree with @akosthekiss, no need to change the snapshot version.

Reply:
@rtakacs two notes:
- the intent of my original comment was to suggest that the version should _not_ be changed (as @LaszloLango also pointed out)
- on these forums, `@akiss` is _not_ me: it's a project/tool ran by INRIA (France). here, I'm `@akosthekiss` ;)